### PR TITLE
fix mavlink codegen enum methods

### PR DESCRIPTION
**Pull request by sirno-baka**
Bug while calling the `MarshalText` method for enum 
Example:
`(common.MAV_SYS_STATUS_SENSOR_3D_GYRO | common.MAV_SYS_STATUS_SENSOR_3D_ACCEL).MarshalText()`

an error occurs because this value is not in labels_{{ENUM}}

I propose to return the hex value of the number.

### CODE REVIEW

**Comment by sirno-baka** [comment]
> specifically requested by some users in order to convert strings into enums

If you need names, I can suggest this code:
```
type MAV_SYS_STATUS_SENSOR uint32

const (
	MAV_SYS_STATUS_SENSOR_3D_GYRO  MAV_SYS_STATUS_SENSOR = 1 << 0 // 0x01
	MAV_SYS_STATUS_SENSOR_3D_ACCEL MAV_SYS_STATUS_SENSOR = 1 << 1 // 0x02
)

var labels_MAV_SYS_STATUS_SENSOR = map[MAV_SYS_STATUS_SENSOR]string{
	MAV_SYS_STATUS_SENSOR_3D_GYRO:  "MAV_SYS_STATUS_SENSOR_3D_GYRO",
	MAV_SYS_STATUS_SENSOR_3D_ACCEL: "MAV_SYS_STATUS_SENSOR_3D_ACCEL",
}

func (e MAV_SYS_STATUS_SENSOR) MarshalText() string {
	var names []string
	for mask, label := range labels_MAV_SYS_STATUS_SENSOR {
		if e&mask == mask {
			names = append(names, label)
		}
	}
	return strings.Join(names, " | ")
}

func (e *MAV_SYS_STATUS_SENSOR) UnmarshalText(text []byte) error {
	labels := strings.Split(string(text), " | ")
	var mask MAV_SYS_STATUS_SENSOR
	for _, label := range labels {
		found := false
		for value, l := range labels_MAV_SYS_STATUS_SENSOR {
			if l == label {
				mask |= value
				found = true
				break
			}
		}
		if !found {
			return fmt.Errorf("invalid label '%s'", label)
		}
	}
	*e = mask
	return nil
}

func (e MAV_SYS_STATUS_SENSOR) String() string {
	return e.MarshalText()
}

```
Tests:

```
value := MAV_SYS_STATUS_SENSOR_3D_GYRO | MAV_SYS_STATUS_SENSOR_3D_ACCEL
fmt.Println(value.MarshalText()) // "MAV_SYS_STATUS_SENSOR_3D_GYRO | MAV_SYS_STATUS_SENSOR_3D_ACCEL"

value = MAV_SYS_STATUS_SENSOR_3D_GYRO
fmt.Println(value.String()) // "MAV_SYS_STATUS_SENSOR_3D_GYRO"

var sens MAV_SYS_STATUS_SENSOR
err := sens.UnmarshalText([]byte("MAV_SYS_STATUS_SENSOR_3D_GYRO | MAV_SYS_STATUS_SENSOR_3D_ACCEL"))
if err != nil {
// handle error
}
fmt.Println(sens.String()) // "MAV_SYS_STATUS_SENSOR_3D_GYRO | MAV_SYS_STATUS_SENSOR_3D_ACCEL"
```

**Comment by sirno-baka** [comment]
It seems that with the update of mavlink, new Enums and Messages have been added. 
#https://github.com/mavlink/mavlink/pull/1943

**Comment by aler9** [comment]
merged, thanks!

File: pkg/dialects/common/enum_adsb_emitter_type.go [hunk 1]
```diff
@@ -57,27 +58,38 @@ var labels_ADSB_EMITTER_TYPE = map[ADSB_EMITTER_TYPE]string{
 
 // MarshalText implements the encoding.TextMarshaler interface.
 func (e ADSB_EMITTER_TYPE) MarshalText() ([]byte, error) {
-	if l, ok := labels_ADSB_EMITTER_TYPE[e]; ok {
-		return []byte(l), nil
+	var names []string
```
Comment (willschenck) [inline pkg/dialects/common/enum_adsb_emitter_type.go:61]: Why was this changed?  This enum is not a bit mask, so these changes to the MarshalText and UnmashalText methods return all the types whose values match the mask, not the single expected string.
For example, I would expect this test to pass:
```			assert.Equal(t, "ADSB_EMITTER_TYPE_ROTOCRAFT", common.ADSB_EMITTER_TYPE_ROTOCRAFT.String())
```
But it fails:
```
-ADSB_EMITTER_TYPE_ROTOCRAFT
+ADSB_EMITTER_TYPE_ROTOCRAFT | ADSB_EMITTER_TYPE_SMALL | ADSB_EMITTER_TYPE_HIGH_VORTEX_LARGE | ADSB_EMITTER_TYPE_HEAVY | ADSB_EMITTER_TYPE_LARGE | ADSB_EMITTER_TYPE_LIGHT | ADSB_EMITTER_TYPE_HIGHLY_MANUV | ADSB_EMITTER_TYPE_NO_INFO
```

**Pull request merged**